test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1691,9 +1696,24 @@ def test_multiple_job_preemption_order(self):
         # Delete one running, one suspended job and one of high priority job
         # This will leave 2 free nodes to pick up the suspended and queued job
 
-        self.server.deljob(jid1)
-        self.server.deljob(jid2)
-        self.server.deljob(jid5)
+        self.server.deljob([jid1, jid2, jid5])
+
+        workq_dict = self.server.status(QUEUE, id='workq')[0]
+        expressq_dict = self.server.status(QUEUE, id='expressq')[0]
+
+        x = 1
```

Comment:
Add a comment to why you're doing this loop.  That deljob(wait=True) will run a scheduling cycle even with scheduling=f.  You want to run jid3 and jid6 to run/resume in the same cycle.  You might also consider submitting a PTL framework bug that deljob() does this.